server/src/router/admin.ts

```diff
@@ -36,7 +36,12 @@ router.put('/:id', validateUser('Admin'), (req, res) => {
     params: { id },
     body,
   } = req;
-  db.update(res, Admin, { id }, body, tableName);
+
+  const Update = () => {
+    db.update(res, Admin, { id }, body, tableName);
```

Comment:
Thanks for noticing this. This is a problem with dynamoose: https://github.com/dynamoose/dynamoose/issues/259.
If I have to be nitpicky, I would recommend just making the Update an anonymous function, because this function is not used anywhere else, but thanks for resolving this issue!
```
db.getById(res, Admin, id, tableName, (result) => {
  db.update(res, Admin, { id }, body, tableName);
});
```